test/replcompletions.jl

```diff
@@ -161,7 +161,7 @@ c,r,res = test_complete(s)
 
 s = "\\a"
 c, r, res = test_complete(s)
-"\\alpha" in c
+("\\alpha" in c)
```

Comment:
Some how this line was being treated as a docstring... not sure what's up there.